### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MiguelMadero**
Updating the comment since it may lead to confusion and it was backwards. baseURL was deprecated in 2.7, but both, baseURL and rootURL were present in previous versions.

### CODE REVIEW

**Comment by workmanw** [comment]
I would say that you should try to keep the version numbers in there, even if you do reword it to make it more clear. Just a personal preference because it's definitive and much easier to just look at than have to jump off and read some other article.

**Comment by ef4** [comment]
Thanks for finding and fixing the misleading backwardness here. I merged the PR and then incorporated @workmanw's suggestion to also keep the version number.

**Comment by MiguelMadero** [comment]
thanks!
